webservices/common/models/itemized.py

```diff
@@ -534,7 +533,10 @@ class ScheduleD(PdfMixin, BaseItemized):
     schedule_type = db.Column(db.String)
     schedule_type_full = db.Column('schedule_type_desc', db.String)
     election_cycle = db.Column(db.Integer)
-    load_date = db.Column('pg_date', db.Date)
```

Comment:
I removed load_date based on feedback from this [PR](https://github.com/fecgov/openFEC/pull/5411).